### ISSUE TITLE
 ERROR    [mvt.ios.modules.backup.configuration_profiles] 

### ISSUE THREAD

**Issue by tocardos** [mvt-project/mvt#216]
hi,

I have an issue with the following script configuration_profiles from ios backup:
```
 ERROR    [mvt.ios.modules.backup.configuration_profiles] Error in      
                  running extraction from module ConfigurationProfiles: Invalid 
                  file                                                          
                  Traceback (most recent call last):                            
                    File "/home/*/.local/lib/python3.8/site-packages/mv
                  t/common/module.py", line 152, in run_module                  
                      module.run()                                              
                    File "/home/*/.local/lib/python3.8/site-packages/mv
                  t/ios/modules/backup/configuration_profiles.py", line 29, in  
                  run                                                           
                      conf_plist = plistlib.load(handle)                        
                    File "/usr/lib/python3.8/plistlib.py", line 1006, in load   
                      raise InvalidFileException()
```
it seems that the module tries to open a file which is not a plist but an XML and returns an error.

I avoided it by adding a test in the run() function :
```
            split_tup= os.path.splitext(conf_file_path)
            if ((split_tup[1]!='.plist') and (split_tup[1]!='.stub')):
                continue
```
I was running the script on a ios 12.3.1 itunes backup. This doesn't affect the overal result.

**Comment by Te-k** [comment]
Hi, thanks for reporting this bug. Can you share the name of the profile file? This file is an XML one and not plist?

**Comment by DonnchaC** [comment]
This should be fixed as 3215e79 changed the module to only parse configuration profiles.
